### ISSUE TITLE
Implement "Recommended" feed.

### ISSUE THREAD

**Issue by tcatut** [bromix/plugin.video.youtube#228]
I was getting really annoyed at not having the "Recommended" feed anymore.  I decided to take a shot at basic implementation of "Activities" from API v3 to make this work again.  My contribution changes are pretty simple so far, but they do work for the "Recommended" feed, and it is using proper API v3 calls which could easily be expanded.

Any chance you can allow me to contribute a pull request? I just tried to submit a pull request myself, and keep getting 403 messages in PyCharm.  I'm thinking you probably have contribution limited to specific people.

Thanks,
Kelly Shutt

**Comment by bromix** [comment]
No, everyone can make a PR....seems to be PyCharm...I can take a look...but I'm really sure, that "Recommended" is deprecated - take a look here: https://developers.google.com/youtube/v3/migration-guide#deprecated

>The v3 API does not retrieve a list that only contains videos recommended for the current API user. 

And the solution via the activities list is a pain (already tried it). But be my guest ;)

**Comment by tcatut** [comment]
As per my console output:

remote: Permission to bromix/plugin.video.youtube.git denied to tcatut.
fatal: unable to access 'https://github.com/bromix/plugin.video.youtube.git/': The requested URL returned error: 403

Any recommendations here?  I tried changing to token authentication, but got the same errors.  

As per my first comment, I already have it working and just need to commit.  Activities wasn't actually that hard to implement, in fact you already had a few hooks in there.  I mainly just had to tell it how to handle the objects that get returned.

**Comment by bromix** [comment]
The activities return more than only recommendations. You have to filter and collect the result for 50 items for more than 2 pages. Thats why I abondend the implementation.

**Comment by bromix** [comment]
Try to fork my project...commit your changes and then pull request. This is the workflow. I don't see any forked project of mine in your account.

**Comment by tcatut** [comment]
Pull request is there now.  It includes support for all the types mentioned here.  https://developers.google.com/youtube/v3/docs/activities

It does also appear to work with pagenation, etc. as the "list" method will return nextPageToken as needed.  I inherited from existing code; so it automatically maps this to a "Next Page" entry in the list.
https://developers.google.com/youtube/v3/docs/activities/list

I have tested on the latest Kodi 14 stable release; for both Windows and Linux platforms.  Both appear to be working as expected.
